### PR TITLE
Ibp 4324 angular nav bar 3

### PR DESCRIPTION
**Pull request by lik-83**

### CODE REVIEW

File: src/main/java/org/generationcp/middleware/service/api/workbench/WorkbenchService.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public interface WorkbenchService {
+
+	WorkbenchSidebarCategoryLink getWorkbenchSidebarLinksByCategoryId(Integer workbenchSidebarCategoryLink);
+
+	void close();
```
Comment (clarysabel) [inline src/main/java/org/generationcp/middleware/service/api/workbench/WorkbenchService.java:13]: is this close() required? @lik-83

File: src/main/resources/liquibase/workbench_changelog/17_2_0.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="lkovacic" id="v17.1.0-1">
```
Comment (clarysabel) [inline src/main/resources/liquibase/workbench_changelog/17_2_0.xml:7]: @lik-83 id should be v17.2.0-2 and version 17.2

File: src/main/java/org/generationcp/middleware/service/api/user/UserDto.java [hunk 3]
```diff
@@ -31,6 +31,8 @@
 	private String password;
 	private Set<CropDto> crops;
 	private Set<String> authorities;
+	private String selectedCropName;
```
Comment (clarysabel) [inline src/main/java/org/generationcp/middleware/service/api/user/UserDto.java:34]: @lik-83 why is this required? is it really an attribute of UserDto?

Comment (lik-83) [reply src/main/java/org/generationcp/middleware/service/api/user/UserDto.java:34]: These attributes are used to show information about the last opened project. Maybe, we can think to create an API to update and get that info.

File: src/main/java/org/generationcp/middleware/service/api/workbench/WorkbenchService.java (hunk 1 above)
Comment (lik-83) [reply src/main/java/org/generationcp/middleware/service/api/workbench/WorkbenchService.java]: done!

File: src/main/resources/liquibase/workbench_changelog/17_2_0.xml (hunk 2 above)
Comment (lik-83) [reply src/main/resources/liquibase/workbench_changelog/17_2_0.xml]: Done!

File: src/main/java/org/generationcp/middleware/service/api/user/UserDto.java (hunk 3 above)
Comment (clarysabel) [reply src/main/java/org/generationcp/middleware/service/api/user/UserDto.java:34]: created: https://ibplatform.atlassian.net/browse/IBP-4397
